### PR TITLE
Add a popup in the GUI if trying to start a match when rocket league is opened without -rlbot flag

### PR DESCRIPTION
**Pull request by swz-gh**
This is a really common problem I see in #issues-and-bugs in the discord server. Instead of users needing to join the discord and ask for help they just see this popup.

screenshot

### CODE REVIEW

File: rlbot_gui/gui/js/main-vue.js [hunk 1]
```diff
@@ -747,6 +751,13 @@ export default {
 			eel.get_recommendations()(recommendations => this.recommendations = recommendations);
 
 			const self = this;
+
+			eel.expose(noRLBotFlagPopup)
+			function noRLBotFlagPopup(title, text){
+				console.log("POPPING UP")
```
Comment (Darxeal) [inline rlbot_gui/gui/js/main-vue.js:757]: Probably forgot to remove this :)

**Review by Darxeal — COMMENTED** [review]
I think there should be a button on the modal. Either a simple "Ok" that closes the modal, or a "Try again" button that that also tries to start the match again.

**Comment by swz-gh** [comment]
Sounds like a good idea, i'll add that.

**Review by Darxeal — APPROVED** [review]
Looks good, thanks!

**Pull request merged**